source/documentation/virtual_machine_management_guide/topics/Configuring_Time_Zones.adoc

```diff
@@ -0,0 +1,20 @@
+[[Configuring_timezones]]
+= Configuring the time zone
+
+{virt-product-fullname} stores time zone configurations for virtual machines in */etc/ovirt-engine/conf/00-timezone.properties*. This file contains default time zone values such as `Etc/GMT=Greenwich Standard Time`. It features mappings that are valid for windows and non-Windows time zones.
```

Comment:
File names are usually written `name`, not *name*. See https://redhat-documentation.github.io/asciidoc-markup-conventions/. I may be missing something, of course, but I'd suggest checking this throughout your text.

Reply:
You're right. There are a lot of occurrences like this throughout the documentation. Legacy issues. Fix them as you come across them.

Reply:
Fixed in this file.